plugins/VisitsSummary/Controller.php

```diff
@@ -40,22 +40,22 @@ public function __construct(Translator $translator)
     /**
      * @deprecated used to be a widgetized URL. There to not break widget URLs
      */
-    public function getSparklines()
+    public function index()
```

Comment:
had them first removed but be good to keep the widget urls working when it's that easy